### ISSUE TITLE
ES6 imports not emitted in declaration files

### ISSUE THREAD

**Issue by csnover** [Microsoft/TypeScript#2220]
Source:

```ts
// foo.ts
var foo = {};
export = foo;
```
```ts
// bar.ts
import foo from './foo';
export = foo;
```

Command line: `tsc -m amd -t es5 -d foo.ts bar.ts`

Expected output of bar.d.ts:

```ts
import foo from './foo';
export = foo;
```

Actual output of bar.d.ts:

```ts
export = foo;
```

Legacy `import foo = require('./foo')` still is emitted.

Version: Arnavion/typescript-github#2015-03-05-b784a4212aeb5a4e42a598a8613a0538ac241123

**Comment by csnover** [comment]
Maybe this is fixed already by #2197, I haven’t tried merging that and testing.

**Comment by csnover** [comment]
Er, no, just merged and tested, it appears to be not resolved by #2197.

**Comment by csnover** [comment]
This is fixed when #2139 lands.

**Comment by danquirk** [comment]
Going to leave this open to ensure it gets checked as fixed when that work is checked in.

**Comment by mhegazy** [comment]
Fixed by #2139
